docs/it/differenze_oidc_fed.rst

```diff
@@ -6,7 +6,7 @@
 Differenze con OIDC Federation
 ------------------------------
 
-In questa sezione sono elencate le differenze che intercorrono tra lo standard ufficiale e l’implementazione SPID.
+In questa sezione sono elencate le differenze che intercorrono tra lo standard ufficiale e l'implementazione SPID.
```

Comment:
Aggiungere anche CIE. Qui le differenze sono tra lo standard di riferimento e la nostra (SPID/CIE) implementazione

Reply:
cambiato in:
In questa sezione sono elencate le differenze che intercorrono tra lo standard ufficiale e l'implementazione SPID / CIE.